acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -76,3 +77,22 @@ def pxp_agent_config_file(host)
     "#{PXP_CONFIG_DIR_CYGPATH}pxp-agent.conf" :
     "#{PXP_CONFIG_DIR_POSIX}pxp-agent.conf"
 end
+
+def configure_pcp_broker(host,instance)
+  set_broker_port(host,instance)
+  set_broker_nrepl_port(host,instance)
+end
+
+def set_broker_port(host,instance)
+  broker_config_path = "#{GIT_CLONE_FOLDER}/pcp-broker#{instance}/test-resources/conf.d/webserver.conf"
+  port_config_name  = 'ssl-port'
+  broker_port = PCP_BROKER_PORTS[instance]
+  on(host, "sed --in-place'' --regexp-extended 's/#{port_config_name}:\\s?[0-9]+/#{port_config_name}: #{broker_port}/' #{broker_config_path}")
```

Comment:
i agree.  i think it's a balance between adding more dependencies and the relative ease/safety of hand-parsing this.